client/src/components/TitleInfo.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+class TitleInfo extends React.Component {
+  render () {
+    const current = this.props.home;
+    return (
+      <div>
+        <p>{current.homeCategory}</p>
+        <p>{current.bedCount} beds</p>
+        <p>{current.starCount}</p>
+        <p>({current.reviewCount})</p>
```

Comment:
Extra parentheses?

Reply:
The number of reviews actually shows up in parentheses on the site so that's there on purpose!